components/floatingactionbutton/index.css

```diff
@@ -79,15 +79,15 @@
 		background-color: var(--highcontrast-floating-action-button-background-color-hover, var(--mod-floating-action-button-background-color-hover, var(--spectrum-floating-action-button-background-color-hover)));
 
 		.spectrum-FloatingActionButton-icon {
-			fill: var(--highcontrast-floating-action-button-icon-color-hover, var(--mod-floating-action-button-icon-color-hover, var(--spectrum-floating-action-button-icon-color-hover)));
+			color: var(--highcontrast-floating-action-button-icon-color-hover, var(--mod-floating-action-button-icon-color-hover, var(--spectrum-floating-action-button-icon-color-hover)));
```

Comment:
Changing from `fill` to `color` made the icon actually follow the defined colors in this CSS file, instead of picking up just the `.spectrum-Icon` definitions.
<img width="122" alt="Screenshot 2025-05-02 at 12 27 16 PM" src="https://github.com/user-attachments/assets/982b2b97-8a93-407c-9a19-f4922e714c8e" />
<img width="152" alt="Screenshot 2025-05-02 at 12 27 07 PM" src="https://github.com/user-attachments/assets/e4967bf8-b00e-4e64-a1ed-8abeccbb22b2" />